ui/src/app/app.tsx

```diff
@@ -47,9 +48,14 @@ const navItems = [
         iconClassName: 'fa fa-user-circle'
     },
     {
-        title: 'Read the documentation, and get help and assistance.',
+        title: 'Documentation',
         path: '/help',
         iconClassName: 'argo-icon-docs'
+    },
+    {
+        title: 'Login',
```

Comment:
I would move login/logout back to where it was. I accidentally logout several times because expand and login buttons are very close and look similar. Also we need to conditionally render login or logout button depending if user has active session or not. 

![image](https://user-images.githubusercontent.com/426437/145655755-c86545f0-22ee-4afa-847d-9f5046a83192.png)